src/util/config.cpp

```diff
@@ -100,6 +101,7 @@ void SystemConfig::print()
     SPDLOG_INFO("--- Scheduling ---");
     SPDLOG_INFO("NO_SCHEDULER               {}", noScheduler);
     SPDLOG_INFO("OVERRIDE_CPU_COUNT         {}", overrideCpuCount);
+    SPDLOG_INFO("USE_TOPOLOGY_HINTS         {}", useTopologyHints);
```

Comment:
As this is switching off something we want on by default, i would change this to `NO_TOPOLOGY` and `noTopology`.